### ISSUE TITLE
Moving data link from GDrive to figshare

### ISSUE THREAD

**Issue by emdann** [InfProbSciX/gplvm_scrna#1]
Hi @InfProbSciX 
To keep this alive for posterity, could you change the google drive link in the README to this figshare link? https://figshare.com/articles/dataset/scRNA_GPLVM_data/24799653 

The GDrive account hosting the data will be deactivated very soon.

**Comment by InfProbSciX** [comment]
Yep absolutely (and sorry for the delayed response). Do you know of a way to download this data via CLI via `wget` or something else (to update the notebooks)? With google cloud, we were using `gdown`.

**Comment by emdann** [comment]
yes, that should be possible with 
```
wget https://figshare.com/ndownloader/articles/24799653/versions/1
```
